plugins/plugin-bash-like/src/lib/util/git-support.ts

```diff
@@ -47,52 +43,19 @@ export const injectCSS = async () => {
  * Return git status, usually for consumption by a git commit
  *
  */
-export const status = (commentPrefix = '# ') => new Promise((resolve, reject) => {
-  const status = exec('git status', (err, status, stderr) => {
-    if (err) {
-      console.error(stderr)
-      reject(err)
-    } else {
-      debug('status', status)
-      const commentedStatus = status
-        .split(/\n/)
-        .map(line => `${commentPrefix}${line}`)
-        .join('\n')
-
-      resolve(commentedStatus)
-    }
-  })
-})
+export const status = () => new Promise(() => {})
 
 /**
  * Find the .git "toplevel" directory
  *
  */
-export const toplevel = (): Promise<string> => new Promise((resolve, reject) => {
-  const status = exec('git rev-parse --show-toplevel', (err, toplevel, stderr) => {
-    if (err) {
-      console.error(stderr)
-      reject(err)
-    } else {
-      resolve(toplevel.trim())
-    }
-  })
-})
+export const toplevel = (): Promise<string> => new Promise(() => {})
 
 /**
  * @return the current branch
  *
  */
-export const branch = (): Promise<string> => new Promise((resolve, reject) => {
-  const status = exec('git rev-parse --abbrev-ref HEAD', (err, branch, stderr) => {
-    if (err) {
-      console.error(stderr)
-      reject(err)
-    } else {
-      resolve(branch.trim())
-    }
-  })
-})
+export const branch = (): Promise<string> => new Promise(() => {})
 
```

Comment:
If these functions are deprecated, we should delete them and remove all the import of them.

Are these functions left on purpose? @starpit

Reply:
let's keep these functions, we may need them in the near future